src/renderer/data/accounts/getAccount.ts

```diff
@@ -51,20 +58,21 @@ export async function getAccount(
 
   const solConn = new sol.Connection(netToURL(net));
   const key = new sol.PublicKey(pubKey);
-  const solAccount = await solConn.getAccountInfo(key);
+  // const solAccount = await solConn.getAccountInfo(key);
+  const solAccount = await solConn.getParsedAccountInfo(key);
```

Comment:
mm, i'll have to see how this plays with the new `getMultipleAccountInfo` method ... that RPC endpoint accepts a jsonparsed arg, so _theoretically_ I see now reason why it wouldn't get parsed, but there's no `getParsedMultipleAccountInfo` in web3.js :|

Reply:
sigh, yea, I can get parsed result from `getMultipleAccountsInfo`, but it blows up on some type of struct validation they do after it returns.

<img width="733" alt="Screen Shot 2022-07-07 at 1 57 31 PM" src="https://user-images.githubusercontent.com/1476820/177870171-fbdd2b6e-dc4c-4c29-b10c-f5816152c6fe.png">
<img width="928" alt="Screen Shot 2022-07-07 at 1 57 41 PM" src="https://user-images.githubusercontent.com/1476820/177870176-183f601f-b37b-4199-bffb-7c06af327be1.png">

To get a parsed version of `getMultipleAccounts`, I'm afraid we might have to duplicate the underlying web3.js code (cringe). Which seems relatively doable by calling those underlying methods like `_buildArgs` fwiw. Though I can't say exactly what makes the result unsafe and if that's a type safety or a security safety concern.

Alternatively -- since for now I think we mostly want to have decoded accounts for -- we could have change the 'legacy' `getAccount` interval in `AccountView` to be a slower poll that calls `getParsedAccountInfo` for the specific account that's being highlighted -- that would unblock this for now 

I did kinda want to display token information within a smaller embedded view in accounts list though :| 

I'm gonna make an issue to track this

Reply:
https://github.com/workbenchapp/solana-workbench/issues/177